main.js

```diff
@@ -36,19 +34,59 @@ function show(event) {
   event.classList.remove('hidden');
 }
 
+function getCooking() {
+  event.preventDefault();
+  hide(cookpotImage);
+  hide(favsList);
+  hide(backButton);
+  show(youShouldMake);
+  show(outputRecipe);
+  show(favButton);
+  show(clearButton);
+
+  getRecipe();
+}
+
 function getRecipe() {
   if (sideOption) {
-    outputRecipe.innerText = `${sidesRecipes[getRandomIndex(sidesRecipes)]}`;
+    currentRecipe = sidesRecipes[getRandomIndex(sidesRecipes)];
   }
   if (mainDishOption) {
-    outputRecipe.innerText = `${mainDishRecipes[getRandomIndex(mainDishRecipes)]}`;
+    currentRecipe = mainDishRecipes[getRandomIndex(mainDishRecipes)];
   }
   if (dessertOption) {
-    outputRecipe.innerText = `${dessertRecipes[getRandomIndex(dessertRecipes)]}`;
+    currentRecipe = dessertRecipes[getRandomIndex(dessertRecipes)];
   }
-  if (entireMealOption) {
-    outputRecipe.innerText = `${mainDishRecipes[getRandomIndex(mainDishRecipes)]} with a side of ${sidesRecipes[getRandomIndex(sidesRecipes)]} and ${dessertRecipes[getRandomIndex(dessertRecipes)]} for dessert!`;
+
```

Comment:
Refactoring of the getRecipe function to use currentRecipe as a placeholder in case the recipe needs to be added to favorites.